scipy/stats/_continuous_distns.py

```diff
@@ -2900,6 +2900,186 @@ def _entropy(self, c):
 genhalflogistic = genhalflogistic_gen(a=0.0, name='genhalflogistic')
 
 
+class genhyperbolic_gen(rv_continuous):
+    r"""A generalized hyperbolic continuous random variable.
+
+    %(before_notes)s
+
+    See Also
+    --------
+    t, norminvgaus, geninvgauss, laplace, cauchy
+
+    Notes
+    -----
+    The probability density function for `genhyperbolic` is:
+
+    .. math::
+
+        f(x, \lambda, \hat{\alpha}, \hat{\beta)} =
+            \frac{(\hat{\alpha}^2 - \hat{\beta}^2)^{\lambda/2}}
+            {\sqrt{2\pi}\hat{\alpha}^{\lambda-0.5}
+            K_\lambda\Big(\sqrt{\hat{\alpha}^2 - \hat{\beta}^2}\Big)}
+            e^{\hat{\beta}(x)} \times \frac{K_{\lambda - 1/2}
+            (\hat{\alpha} \sqrt{1 + x^2})}
+            {(\sqrt{1 + x^2})^{1/2 - \lambda}}
+
+    for :math:`x, \lambda \in \mathbb{R}`,
+    :math:`|\hat{\beta}| \lt \hat{\alpha}` if :math:`\lambda \ge 0`,
+    :math:`|\hat{\beta}| \lt \hat{\alpha}` if :math:`\lambda = 0`,
+    :math:`|\hat{\beta}| \leq \hat{\alpha}` if :math:`\lambda \lt 0`.
+    :math:`K_{\lambda}(.)` denotes the modified Bessel function of the second
+    kind and order :math:`\lambda` (`scipy.special.kn`)
```

Comment:
This is showing as:
![image](https://user-images.githubusercontent.com/6570539/104498691-e58ff300-5590-11eb-80a0-4c4720f6c03d.png)

Looks like the second requirement on `\hat{\beta}` is redundant, right? The first is for `\lambda \geq 0`, so that includes `=0`. 
Also, is `\hat \alpha` allowed to be negative? If so, how can the magnitude of `\hat \beta` be less than it?

I don't mind personally, but many often remind me that users read these in the console. It might be simpler for those users - and it would align with the code - if you were to replace `\hat \alpha` with `a` and `\hat \beta` with `b`. You could even consider replacing `\lambda` with `p`, but that is up to you.

Reply:
Thank you for spotting! Actually the first condition that should be  `\gt` instead of `\ge` . In fact https://www.jstor.org/stable/4615705 defines existence conditions on `\delta `as well (which can be 0 when `\lambda \gt 0`), but because scipy hard checks `scale `to be strictly `\gt 0` i agree first 2 conditions conditions can be collapsed into a single one. Wil fi that in the tutorial as well.

I'd rather leave lambda and the hats because i write just after this paragraph about the conversion latin-greek letters. Will ponder if it is the case to change anyway.

Reply:
Right but the conversion could just be:
`a = \alpha \delta`, right?

Reply:
That is also a valid option. I went with the double conversion because the hat is the notation in https://doi.org/10.1007/978-3-662-12429-1_12 (reference for the loc-scale parameterization), but i see how it is a bit redundant

Reply:
I'm glad you ended up simplifying these to _a_, _b_, and _p_! It's a good compromise between those who look at the documentation in browser vs in console.